functions/wcif.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs')
+
+const ClearWCIF = {
+  name: 'ClearWCIF',
+  // FIXME: quick helper to make sure we start "from scratch", maybe also
+  // clean other properties/extensions which I'm not aware of?
+  docs: 'Remove all childActivities keeping only the main schedule, remove all assignments.',
+  args: [],
+  outputType: 'Void',
+  usesContext: true,
+  mutations: ['schedule', 'persons'],
+  implementation: (ctx, filename) => {
+    ctx.competition.persons.forEach((person) => {
+      // FIXME: should we clean up "roles" too actually?
+      // Changing "delegate"/"organizers"/"trainee-delegate" would have no effect
+      // as they are - expectedly! - not patchable on the WCA, but any "staff-*"
+      // should likely be removed too.
+      person.assignments = []
+    })
+    ctx.competition.schedule.venues.forEach((venue) => {
+      venue.rooms.forEach((room) => {
+        room.extensions = []
+        room.activities.forEach((activity) => {
+          activity.extensions = []
+          activity.childActivities = []
+        })
+      })
+    })
+  }
+}
+
+const ImportWCIF = {
+  name: 'ImportWCIF',
+  docs: 'Import the WCIF from a json file',
+  args: [
+    {
+      name: 'filename',
+      type: 'String',
+      docs: 'WCIF filename (relative to SCRIPT_BASE)',
```

Comment:
Why is ImportWCIF relative to SCRIPT_BASE, and ExportWCIF relative to OUTPUT_BASE?

Reply:
Initially my thought process was that it would make sense to have all the data required to (re)create the assignments at the same place.
But maybe we're fine with just having a "WCIF_DATA_FOLDER" directory serving as a base for all import/export operations, and where we let the user organize it how ever they want.
In which case I would drop the commit about always setting up `SCRIPT_BASE`, and I would rename `OUTPUT_BASE` to `WCIF_DATA_FOLDER` (or anything else clearly indicating what we aim to store there :p); what do you think?

Reply:
Fixed by making the import/export relative to `WCIF_DATA_FOLDER`.